vip-dashboard/components/widget-welcome.php

```diff
@@ -7,6 +7,35 @@ function render_vip_dashboard_widget_welcome() {
 
 		<p>WordPress VIP is a partnership between Automattic and the most high-profile, innovative and smart WordPress websites out there. We’re excited to have you here.</p>
 
+		<h3 class="widget__subtitle">WPVIP Dashboard Links</h3>
+
+		<a href="https://dashboard.wpvip.com/apps/<?= VIP_GO_APP_ID . "/" . VIP_GO_APP_ENVIRONMENT ?>" target="_blank">WPVIP Dashboard</a>
```

Comment:
I don't think this will work as intended for child sites since `VIP_GO_APP_ID` doesn't translate to the parent ID.